examples/community-detection.ipynb

```diff
@@ -0,0 +1,1148 @@
+       "      <th>modificationTime</th>\n",
+       "      <th>schema</th>\n",
+       "      <th>schemaWithOrientation</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>{'p99': 15, 'min': 0, 'max': 87, 'mean': 1.631...</td>\n",
+       "      <td>reddit</td>\n",
+       "      <td>neo4j</td>\n",
+       "      <td>876 KiB</td>\n",
+       "      <td>897448</td>\n",
+       "      <td>3801</td>\n",
+       "      <td>6200</td>\n",
+       "      <td>{'relationshipProjection': {'HYPERLINKED_TO': ...</td>\n",
+       "      <td>0.000429</td>\n",
+       "      <td>2023-05-14T15:29:30.028518203+00:00</td>\n",
+       "      <td>2023-05-14T15:29:30.931965175+00:00</td>\n",
+       "      <td>{'graphProperties': {}, 'relationships': {'HYP...</td>\n",
+       "      <td>{'graphProperties': {}, 'relationships': {'HYP...</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "                                  degreeDistribution graphName database  \\\n",
+       "0  {'p99': 15, 'min': 0, 'max': 87, 'mean': 1.631...    reddit    neo4j   \n",
+       "\n",
+       "  memoryUsage  sizeInBytes  nodeCount  relationshipCount  \\\n",
+       "0     876 KiB       897448       3801               6200   \n",
+       "\n",
+       "                                       configuration   density  \\\n",
+       "0  {'relationshipProjection': {'HYPERLINKED_TO': ...  0.000429   \n",
+       "\n",
+       "                          creationTime                     modificationTime  \\\n",
+       "0  2023-05-14T15:29:30.028518203+00:00  2023-05-14T15:29:30.931965175+00:00   \n",
+       "\n",
+       "                                              schema  \\\n",
+       "0  {'graphProperties': {}, 'relationships': {'HYP...   \n",
+       "\n",
+       "                               schemaWithOrientation  \n",
+       "0  {'graphProperties': {}, 'relationships': {'HYP...  "
+      ]
+     },
+     "execution_count": 8,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "gds.graph.list()"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "9c259471",
+   "metadata": {},
+   "source": [
+    "### Weakly Connected Components\n",
+    "\n",
+    "A graph dataset need not always be connected. That is, there may not exist a path from every node to \n",
+    "every other node in the graph dataset (subgraphs in it may not connected to each other at all). Hence, we \n",
+    "need to find the total number of nodes in each subgraph to see if it is big enough for further graph analysis. \n",
+    "Smaller subgraphs or lone nodes will not contribute to the community detection task and should be \n",
+    "eliminated. Weakly Connected Components is often used as one of the early steps of graph preprocessing.\n",
```

Comment:
Could explanation :) 
FYI - Also we offer a `minCommunitySize` parameter in our community algorithm.
In the latest release its only available on `louvain.write` but in the upcoming `2.4` it will be also be possible in stream and write.

But for now this is workflow is good and in the future with 2.4 we can add a section for this case.